src/components/App.tsx

```diff
@@ -263,7 +263,7 @@ class App extends TsxComponent<AppProps> {
     }
     // We only want to generate the content, when the app is correctly initialized
     const content =
-      this.appState === FSXAAppState.ready ? this.renderContent() : [];
+      this.appState === FSXAAppState.ready ? this.renderContent() : null;
```

Comment:
Doesn't it change the type of the appState?

Reply:
This is in response to a Vue console warning that kept popping up during the test runs (in every test case). `[]` implies multiple Vue elements would be rendered and it seems to me this is not allowed at the root level of a component, as is the case in React, so I replaced it with null which was probably the intention. I am not sure why `[]` was the choice here in the first place.